### PR TITLE
outdent err block

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```NONE

```

### CODE REVIEW

File: pkg/volume/util.go
Comment (jhorwit2) [inline pkg/volume/util.go]: I think you forgot an `else` here.

**Review by jhorwit2 — CHANGES_REQUESTED** [review]
Thanks for the PR @zouyee! I think you have a syntax error but overall LGTM after that!

**Comment by zouyee** [comment]
ping @jhorwit2

File: pkg/volume/util.go
Comment (davidz627) [inline pkg/volume/util.go]: nit: GetPath {checks}

**Comment by davidz627** [comment]
please run hack/update-gofmt.sh on your changed files. I think it is failing the gofmt verify script.

**Comment by zouyee** [comment]
ping @davidz627

File: pkg/volume/util.go
Comment (davidz627) [inline pkg/volume/util.go]: nit: lowercase *checks*

File: pkg/volume/util.go
Comment (davidz627) [inline pkg/volume/util.go]: nit: remove space

File: pkg/volume/util.go
Comment (davidz627) [inline pkg/volume/util.go]: nit: remove space.

Comment (zouyee) [reply pkg/volume/util.go]: It has been fixed, thanks again.

**Comment by zouyee** [comment]
ping @gnufied

**Pull request merged**
